### ISSUE TITLE
SYSTEM_CONTAINER

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#19080]
A special container that will accept any and all content types, the system container will be shipped in the empty starter, vtls files for the system container will be in the **default** host under`/application/containers/system`.

- [x] We will have a couple of`.vtls` files that live in `application/containers/system`:

- **container.vtl**
```
$dotJSON.put("title", "System Container")
$dotJSON.put("max_contentlets", 25)
$dotJSON.put("notes", "System Container")
$dotJSON.put("useDefaultLayout", "*") //list of content types accepted by the container. * means all. 
```
- [x] The list of content types that are available for adding should be permission-based on the user
- **default_container.vtl** will be used to format all content types that don't have a .vtl
```
<h3>$title<h3>
``` 
- [x] It will be appended to the list of available containers
- [x] Add `vtls` to empty starter.

**Comment by erickgonzalez** [comment]
PR: https://github.com/dotCMS/core/pull/19456

**Comment by erickgonzalez** [comment]
Steps to test:
1- Startup dotcms using the empty stater.
2- Create a new Advanced Template and add the System Container.
3- Create a new contentlet of any content type.
4- Create a new page using the template created above.
5- Add the contentlet to the created page.

You can modify the **container.vtl** the property ** useDefaultLayout** to a list of content types instead of * that means all content types, so the system container only accepts those content types.

**Comment by bryanboza** [comment]
Fixed, tested on release-21.04 // Postgres // FF. Now we have the system container in the empty starter and we are able to use without problems
